Document the Plan & Usage page for partners (#34673)
DataDog/documentation: content/en/account_management/plan_and_usage/partner_experience.md

---
title: Plan and Usage Experience for Partners
description: Explains how Datadog partners can use the Plan and Usage page to view cost and usage data across customer organizations from a partner admin organization.
further_reading:
- link: "https://docs.datadoghq.com/account_management/plan_and_usage/"
  tag: "Documentation"
  text: "Plan & Usage"
- link: "https://docs.datadoghq.com/account_management/plan_and_usage/cost_details/"
  tag: "Documentation"
  text: "Cost Details"
- link: "https://docs.datadoghq.com/account_management/plan_and_usage/usage_details/"
  tag: "Documentation"
  text: "Usage Details"
- link: "https://docs.datadoghq.com/api/latest/usage-metering/"
  tag: "Documentation"
  text: "Usage Metering API"
---

Datadog partners see a tailored version of the Plan & Usage page designed for managing multiple customer organizations. This page describes the Plan & Usage experience available to partners viewing the UI from a **partner admin organization**.

## Prerequisites

> To access the partner Plan & Usage experience, your organization must be a partner admin organization, which is the top-level organization that Datadog provisions for partners to manage their customer accounts. If your organization was set up through the Datadog Partner Program and you manage customer organizations under it, you are using a partner admin organization. You can confirm your status by validating that you have multiple contracts with Datadog and can view consolidated cost and usage data across all of them in Plan & Usage.

Reviewer: ```suggestion
To access the partner Plan & Usage experience, your organization must be a partner admin organization, which is the top-level organization that Datadog provisions for partners to manage their customer accounts. If your organization was set up through the Datadog Partner Program and you manage customer organizations under it, you are using a partner admin organization. To verify, check that you have multiple contracts with Datadog and can view consolidated cost and usage data across all of them in Plan & Usage.
```

> **Note**: If you are a *customer* of a Datadog partner (rather than the partner themselves), you have the standard Plan & Usage experience. Depending on your pricing arrangement with your partner, you may see only usage data and no cost data. See [Plan & Usage][1] for details.

Reviewer: ```suggestion
**Note**: If you are a *customer* of a Datadog partner (rather than the partner itself), you have the standard Plan & Usage experience. Depending on your pricing arrangement with your partner, you may see only usage data and no cost data. See [Plan & Usage][1] for details.
```

Partners must have the **Datadog Admin** role, or a custom role with `billing_read` and `usage_read` permissions, to view Plan & Usage data. These are the same permissions required for direct customers.

## Overview

Partners accessing Plan & Usage from their partner admin organization see cost and usage data across all of their customer organizations. Only the **Usage & Cost** tab is available in the partner view; the Plan, Billing History, and Usage Notifications tabs are not displayed.

> ### Cost Details

Reviewer 2: Nit: sentence case for the heading.

```suggestion
### Cost details
```

View estimated, historical, and projected cost data for all customer organizations in one place. Partners can see a total across all customers, or group and filter by customer organization, product, and account. See [Cost Details][2] for full documentation.

{{< img src="account_management/plan_and_usage/partner-cost-details.png" alt="Cost Summary page for a partner admin organization showing estimated and projected costs across customer organizations with a cumulative cost breakdown chart and per-customer cost table." >}}

> ### Usage Details

Reviewer 2: Same nit — sentence case.

```suggestion
### Usage details
```

View usage data for all customer organizations in one place. Partners can see a total across all customers, or group and filter by customer organization, product, and account. See [Usage Details][3] for full documentation.

{{< img src="account_management/plan_and_usage/partner-usage-details.png" alt="Usage summary page for a partner admin organization showing usage data across customer organizations." >}}

## Feature availability

The following table lists each Plan & Usage feature and its availability for direct customers and partners viewing from a partner admin organization.

| Feature | Direct Customers | Partners | Notes |
|---|---|---|---|
| Cost summary | {{< X >}} | {{< X >}} |  |
| Projected costs | {{< X >}} | {{< X >}} |  |
| Usage summary | {{< X >}} | {{< X >}} |  |
| Cost & usage API endpoints | {{< X >}} | {{< X >}} | See [Supported API endpoints](#supported-api-endpoints) for the full list |

> | Cost attribution |  |  |  |

Reviewer 2: Cost attribution is available for direct customers and accessible to individual customer orgs from their own Plan & Usage page.

```suggestion
| Cost attribution | {{< X >}} |  | Individual customer organizations can access cost attribution from their own [Plan & Usage][1] page |
```

| Usage attribution | {{< X >}} |  | Individual customer organizations can access usage attribution from their own [Plan & Usage][1] page |
| Product-specific usage tables (for example, custom metrics, logs usage by index) | {{< X >}} |  | Individual customer organizations can view these tables from their own [Usage Details][3] page |

> | Datadog costs in Cloud Cost Management | {{< X >}} |  | Not available at the partner admin org level or the customer org level for customers purchasing through a partner |

Reviewer 2: Individual customer orgs can access Datadog Costs in CCM from their own CCM page, so let's point them there rather than saying it's unavailable.

```suggestion
| Datadog costs in Cloud Cost Management | {{< X >}} |  | Individual customer organizations can access [Datadog Costs](/cloud_cost_management/datadog_costs/) from their own Cloud Cost Management page |
```

> | Plan tab and Billing History tab |  |  | Not available at the partner admin org level or the customer org level for customers purchasing through a partner |

Reviewer 2: Plan and Billing History tabs are available for direct customers, and individual customer orgs can access them from their own Plan & Usage page.

```suggestion
| Plan and Billing History tabs | {{< X >}} |  | Individual customer organizations can access Plan and Billing History tabs from their own [Plan & Usage][1] page |
```
